scripts/linux/trcE2ETest.sh

```diff
@@ -917,7 +917,7 @@ function configure_connectivity_settings() {
 
     TEST_INFO="$TEST_INFO,TestDuration=${TEST_DURATION}"
 
-    log_upload_enabled=false
+    log_upload_enabled=true
```

Comment:
Should this variable get set back to false before merging the PR?

Reply:
Sure, I'll do it. 

It's much of a muchness at this point though, the upload will still fail, though I guess it's worth not waiting around for it to try and upload.